app/services/ExposureNotificationService.ts

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line
+declare const global: any;
+
+global.ExposureNotificationsOn = true;
```

Comment:
mmm, is there a way we could do this without using global?

Reply:
Yeah - globals aren't great. What did you have in mind? This was a quick and dirty since we knew it was going to come out in a couple of days anyways.

Reply:
chatted irl, might be able to save the state in a context!